userland/examples/tests/adc/main.c

```diff
@@ -0,0 +1,80 @@
+#include <string.h>
```

Comment:
I moved the `adc` app to `examples/tests/`. Is this the correct choice? I'm not sure what the distinguishing feature is between things that are examples and things that are tests.